pkg/controller/pod.go

```diff
@@ -200,19 +200,27 @@ func (pnc *PodNetworksController) processNextWorkItem() bool {
 		return true
 	}
 
-	indexedNetworkSelectionElements := annotations.IndexPodNetworkSelectionElements(pod)
-	indexedNetworkStatus := annotations.IndexNetworkStatusIgnoringDefaultNetwork(pod)
+	networkSelectionElements, _ := annotations.PodNetworkSelectionElements(pod) // ignore error as other functions below?
+	networkStatus, _ := annotations.PodDynamicNetworkStatus(pod)
```

Comment:
I am concerned about this (but not sure if I should be ...).

Couldn't we handle the error (returning early) ? My motivation is if I can't retrieve the pod's network status, I should just avoid handling this pod. Same with the network selection elements - the pod does not comply with "our" API expectations.

Reply:
Right, I fixed it.